tests/test_instances.py

```diff
@@ -0,0 +1,65 @@
+"""Tests for training instances."""
+from typing import Any, MutableMapping
+
+import torch.utils.data
+import unittest_templates
+
+from pykeen.datasets import Nations
+from pykeen.triples import LCWAInstances, SLCWAInstances
+from pykeen.triples.instances import BatchedSLCWAInstances, SLCWABatch
+from tests import cases
+
+
+class LCWAInstancesTestCase(cases.TrainingInstancesTestCase):
+    """Tests for LCWA training instances."""
+
+    cls = LCWAInstances
+
+    def _pre_instantiation_hook(self, kwargs: MutableMapping[str, Any]) -> MutableMapping[str, Any]:  # noqa: D102
+        kwargs = super()._pre_instantiation_hook(kwargs=kwargs)
+        other_instance = self.cls.from_triples(
```

Comment:
why not just override the instantiation hook?

Reply:
this looked like more effort to me.